.circleci/config.yml

```diff
@@ -1,12 +1,14 @@
 defaults: &defaults
   working_directory: ~/thrift-server
   docker:
-    - image: circleci/node:8.9.4
+    - image: circleci/node:12.9.0
```

Comment:
Node 12?

Reply:
Just to make sure we also support node 12 till all services move to node14.. Testing two versions of node..